tests/plugins/pastefromword/manual/unorderedlistspecialchar.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: bug, 4.8.0, 877, pastefromword
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, pastefromword, sourcearea, list
+
+----
+1. Open browser console.
+1. Open [file](../generated/_fixtures/Unordered_list_special_char_bullet/Unordered_list_special_char_bullet.docx) in Word.
```

Comment:
Maybe you can put file name here as a link `[Unordered_list_special_char_bullet.docx](...)`?